includes/controls/media.php

```diff
@@ -128,66 +128,141 @@ public function enqueue() {
 	 */
 	public function content_template() {
 		?>
+		<#
+			// For BC.
+			if ( data.media_type ) {
+				data.media_types = [ data.media_type ];
+			}
+
+			if ( data.should_include_svg_inline_option ) {
+				data.media_types.push( 'svg' );
+			}
+
+			// Determine if the current media type is viewable.
+			const isViewable = () => {
+				const viewable = [
+					'image',
+					'video',
+					'svg',
+				];
+
+				let output = true;
+
+				// Make sure that all media types are viewable.
+				data.media_types.forEach( ( type ) => {
+					if ( ! viewable.includes( type ) ) {
+						output = false;
+						return;
+					}
+				} );
+
+				return output;
+			};
+
+			// Get the preview type for the current media type.
+			const getPreviewType = () => {
+				if ( data.media_types.includes( 'video' ) ) {
+					return 'video';
+				}
+
+				if ( data.media_types.includes( 'image' ) || data.media_types.includes( 'svg' ) ) {
+					return 'image';
+				}
+
+				return 'none';
+			}
+
+			// Retrieve a button label by media type.
+			const getButtonLabel = ( mediaType ) => {
+				switch( mediaType ) {
+					case 'image':
+						return '<?php echo __( 'Choose Image', 'elementor' ); ?>';
+						break;
+
+					case 'video':
+						return '<?php echo __( 'Choose Video', 'elementor' ); ?>';
+						break;
```

Comment:
Redundant `break;`

Reply:
Next time you can just write that once, I will fix the others automatically as well 😄